internal/jstest/jstest.go

```diff
@@ -149,8 +150,17 @@ func runCommand(dir, command string, args ...string) ([]byte, error) {
 		return nil, err
 	}
 
-	args = append([]string{"--verbose", "--norc", "mongodb://host.docker.internal:27017/"}, args...)
+	// creates the TestData variable and sets the testName property for the shell
+	var eb strings.Builder
+	eb.WriteString("TestData = new Object();")
+	eb.WriteRune(' ')
+	f := filepath.Base(args[len(args)-1])
```

Comment:
That looks sus. What is the resulting command-line?

Reply:
I also looked into setting this variable in a `.mongorc.js` file to avoid command-line arguments. This does not work because it only read this file after evaluating what has been passed on the [command line](https://www.mongodb.com/docs/mongodb-shell/mongoshrc/#.mongoshrc-configuration-file).

Reply:
Sorry, but I think it is time to refactor that code. For example, `args` is always a single file, but instead of passing a single string, we continue treating `args` as a slice and modifying it in place, making our code needlessly complicated.

Please make the code in this file simpler.

Reply:
And speaking of simplifying: `command` is always a `mongo` and not even a command – it is the Docker Compose service name. I think that function variable could be removed because it is not variable